pkg/buildkite/access_token_test.go

```diff
@@ -24,13 +25,24 @@ func (m *MockAccessTokenClient) Get(ctx context.Context) (buildkite.AccessToken,
 
 func TestAccessToken(t *testing.T) {
 	assert := require.New(t)
+	testTime := time.Date(2023, 1, 1, 0, 0, 0, 0, time.UTC)
 
 	ctx := context.Background()
 	client := &MockAccessTokenClient{
 		GetFunc: func(ctx context.Context) (buildkite.AccessToken, *buildkite.Response, error) {
 			return buildkite.AccessToken{
-					UUID:   "123",
-					Scopes: []string{"read_build", "read_pipeline"},
+					UUID:        "123",
+					Scopes:      []string{"read_build", "read_pipeline"},
+					Description: "Test token",
+					User: struct {
```

Comment:
Its awesome that we get this sideloaded now!

Reply:
I wonder actually if the current_user tool is quite as necessary, or if the two can be combined somehow? I guess the current_user one needs a different scope